dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -173,23 +171,18 @@ public class JdbcEventStore implements EventStore
     private static final String RELATIONSHIP_IDS_QUERY = " left join (select ri.programstageinstanceid as ri_psi_id, json_agg(ri.relationshipid) as psi_rl FROM relationshipitem ri"
         + " GROUP by ri_psi_id)  as fgh on fgh.ri_psi_id=event.psi_id ";
 
-    private static final String PSI_EVENT_COMMENT_QUERY = "select psic.programstageinstanceid    as psic_id," +
-        " psinote.trackedentitycommentid as psinote_id," +
-        " psinote.commenttext            as psinote_value," +
-        " psinote.created                as psinote_storeddate," +
-        " psinote.creator                as psinote_storedby," +
-        " psinote.uid                    as psinote_uid," +
-        " psinote.lastupdated            as psinote_lastupdated," +
-        " userinfo.userinfoid            as usernote_id," +
-        " userinfo.code                  as usernote_code," +
-        " userinfo.uid                   as usernote_uid," +
-        " userinfo.username              as usernote_username," +
-        " userinfo.firstname             as userinfo_firstname," +
-        " userinfo.surname               as userinfo_surname" +
-        " from programstageinstancecomments psic" +
-        " inner join trackedentitycomment psinote" +
-        " on psic.trackedentitycommentid = psinote.trackedentitycommentid" +
-        " left join userinfo on psinote.lastupdatedby = userinfo.userinfoid ";
+    private static final String PSI_EVENT_COMMENT_QUERY = "select psic.programstageinstanceid    as psic_id,"
+        + " psinote.trackedentitycommentid as psinote_id," + " psinote.commenttext            as psinote_value,"
```

Comment:
Looks like the format got messed up here. Is it possible to fix this?

Reply:
@stian-sandvold I literally have no idea what happened with the formatting. I've restored most of the previous formatting, some new changes were actually a bit better so I've left them